test/test_core.py

```diff
@@ -1953,10 +1953,8 @@ def test_em_js(self, args, force_c):
 
   @no_wasm2js('WASM_BIGINT is not compatible with wasm2js')
```

Comment:
This comment maybe needs updating or removing?

Reply:
Comment updated.

Reply:
I don't see any change here?

Reply:
It's in [a8372fc](https://github.com/emscripten-core/emscripten/pull/22993/commits/a8372fcae092bba23aa1782f62fdac2df1b76ca0)